src/mobX/models/workpieces/documentation/PerformerModel.js

```diff
@@ -7,10 +7,10 @@ import { observable, action, computed } from "mobx"
 import PlayerInstrumentModel from "./PlayerInstrumentModel"
 const makeObservable = () => {}
 export const PerformerType = {
-	mainArtist: "mainArtist",
+	principal: "principal",
```

Comment:
NO, NO, NO! Make your own PerformerType that you use in your model NOT the documentation model, the schemas are NOT the same.

Reply:
return them top the way they were and make your own, please

Reply:
No probs